src/Templating/BaseTemplateRegistry.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Templating;
+
+abstract class BaseTemplateRegistry implements TemplateRegistryInterface
```

Comment:
```suggestion
abstract class AbstractBaseTemplateRegistry implements TemplateRegistryInterface
```

Reply:
AbstractTemplateRegistry will be more clear and enought.